gcp/modules/gcp-stackdriver-lbm/main.tf

```diff
@@ -7,6 +7,7 @@ variable "project_id" {}
 variable "serviceaccount_key" {}
 
 # Enables debug mode when TF_VAR_stackdriver_debug is not empty
+
```

Comment:
Extra newline?

Reply:
Terraform freaks out and complains about broken formatting without this newline.